spec/features/super_admin/add_custom_organisation_spec.rb

```diff
@@ -6,11 +6,20 @@
   let!(:admin_user) { create(:user, super_admin: true) }
 
   context 'when visiting the custom organisations page' do
+    let!(:org2) { CustomOrganisationName.create(name: 'Custom Org 2') }
```

Comment:
someone is definitely going to come and put these in order

Reply:
?

Reply:
Haha